lib/sufia/arkivo/actor.rb

```diff
@@ -0,0 +1,96 @@
+require 'base64'
+require 'tempfile'
+
+# Ugly monkeypatch to make Tempfiles quack like UploadedFiles
+Tempfile.class_eval do
+  attr_accessor :original_filename, :content_type
+end
+
+module Sufia
+  module Arkivo
+    class Actor
+      attr_reader :user, :item
+
+      def initialize(user, item)
+        @user = user
+        @item = item
+      end
+
+      def create_file_from_item
+        batch = Batch.create
+        file_actor = Sufia::GenericFile::Actor.new(::GenericFile.new, user)
+        file_actor.create_metadata(batch.id)
+        store_checksum(file_actor.generic_file)
+        file_actor.create_content(file, item['file']['filename'], file_path, item['file']['contentType'])
+        ::Resque.inline = true
+        Sufia.queue.push(BatchUpdateJob.new(user.user_key, batch.id, item['metadata']['title'], attributes, default_visibility))
```

Comment:
Wait...does the `Sufia.queue.push(BatchUpdateJob.new(...))` not run in the background because of the `::Resque.inline = true` above it? Why push it to the queue in this case? 

Why not just call `BatchUpdateJob.new(...)` directly?